src/server/lib/oauth/callback.js

```diff
@@ -121,9 +110,8 @@ async function _getProfile ({
         provider: provider.id,
         type: provider.type,
         id: profile.id,
-        refreshToken,
-        accessToken,
-        accessTokenExpires: null
+        ...tokens,
+        accessTokenExpires: results.expires_in ?? null
```

Comment:
@iaincollins any reason why the raw result of the tokens fetch call not simply spread onto the `account` object or added as a `tokens` property or something? Currently, It seems to only be remapping of only **some** of those values, plus the `accessTokenExpires` is null'ed out. Also, I don't see the need for `provider.id` being added twice(?) both as `provider` and `id`.